packages/color-scale-block/src/components/ColorPickerFlyout.tsx

```diff
@@ -0,0 +1,110 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
File to be removed

Reply:
Why removed? If you think about `ColorPickerFlyout` from Fondue, it's a completely different component.

Reply:
The file is in `color-scale-block`

Reply:
Oh damn, sry, my bad 😄